sdk/appconfiguration/azure-spring-cloud-appconfiguration-config/pom.xml

```diff
@@ -172,6 +172,18 @@
           </rules>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
```

Comment:
@srnagar 

Does this change missing something. The PR build failed at verifying version.

Reply:
@mssfang This is missing a `<version></version>` tag.

Reply:
@srnagar As the version was listed as managed.

Reply:
Version should still be explicitly specified to not use the managed version and instead use the version we use for all client libraries.

Reply:
`<version>3.1.2</version> <!-- {x-version-update;org.apache.maven.plugins:maven-jar-plugin;external_dependency} -->`